components/private/ShapeLibrary/tests/unit/ShapeLibraryTests.m

```diff
@@ -39,10 +39,12 @@ - (void)testCurvedCornerEquality {
   MDCCornerTreatment *cornerTreatment = [MDCCornerTreatment cornerWithCurve:CGSizeMake(2, 5)];
 
   // When
+  XCTAssertNotEqual(curvedCorner.hash, curvedCorner2.hash);
```

Comment:
Please remove these `NotEqual` asserts. Hashes can be not equal when the objects are equal. It's the reverse (Equality requires identical hashes) that has to be enforced.

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/5954